### ISSUE TITLE
Maintaining this project

### ISSUE THREAD

**Issue by jonhealy1** [stac-utils/qgis-stac-plugin#253]
It looks like this project could use some new maintainers. If anyone is interested leave a comment here. This is an important project and it would be nice to see it updated.

**Comment by captaincoordinates** [comment]
This is an old issue now, but just in case: basic STAC support was [added to QGIS core](https://www.lutraconsulting.co.uk/blogs/stac-in-qgis) in 3.40, extended in 3.42, and will be [further extended](https://github.com/qgis/QGIS/pull/63358) in 4.0. Development of this plugin appears to have stopped and hopefully it is no longer required.

**Comment by arer01** [comment]
I dont understand, how do i use the Stac browser without the plug-in?
I can add the Stac layer but how do filter down so i only see the ones on the screen
And how do the downloading of the assets function

**Comment by captaincoordinates** [comment]
@arer01 in 3.44 basic functionality (list collections & items, drag items with cloud-optimised assets to Layers pane) exists in the Browser pane. More functionality exists in the Data Source Manager, including basic filtering and downloading assets. Support in QGIS core is currently growing and in 4.0 there will be more.

I don't know if QGIS core will eventually do everything that the plug-in did, but STAC in QGIS core is currently maintained while it appears that this plugin is not.

<img width="1409" height="848" alt="Image" src="https://github.com/user-attachments/assets/c78c7f58-9614-41bf-b01f-d07fa374b6e6" />

**Comment by arer01** [comment]
I cant find a filter that only shows STAC that are present i the current geographical position? 
And it says double-click to show more. How can i make it collect all STAC posts at once

**Comment by remicres** [comment]
I agree that there is some room for improvement. But good news that STAC API browsing is now part of QGIS
